src/ui/components/Comments/CommentMarker.js

```diff
@@ -52,15 +52,17 @@ class CommentMarker extends React.Component {
   }
 
   renderCreateCommentButton() {
-    const { createComment, currentTime } = this.props;
+    const { createComment, currentTime, zoomRegion } = this.props;
 
     if (this.getCommentAtTime(currentTime)) {
       return null;
     }
 
     return (
       <button
-        className={classnames("create-comment")}
+        className={classnames("create-comment", {
+          overflowed: currentTime > zoomRegion.endTime,
+        })}
```

Comment:
why not return null above?

the benefit might be that it's fewer divs in the dom?

Reply:
Good catch, thanks. Fixed!